packages/farcaster-js-hub-rest/tests/integration/hubRestAPIClient.ts

```diff
@@ -19,10 +19,17 @@ import {
   UserDataType,
 } from '../../src/openapi/index.js';
 import type { paths as SchemaPaths } from '../../src/openapi/generated/schema.js';
+import { hexToBytes } from '@noble/hashes/utils';
```

Comment:
this line is marked as a relative import, fix added to package.json

Reply:
whoa, I'd be curious to know more